public/index.js

```diff
@@ -2,11 +2,9 @@
 // Ignoring this file since it has to be written in ES5
 // and eslint is configured to lint ES6.
 
-module.exports = function(currentUser, matchingUsers) {
-  var params = window.Qs.parse(window.location.search, { ignoreQueryPrefix: true });
-
+module.exports = function(currentUser, matchingUsers, params, token) {
```

Comment:
out of curiosity this gets executed on page load like before because do `<script src="/index.js"></script>`?

Reply:
it is bundled and all together ends being part of [bundle.js](https://github.com/auth0-extensions/auth0-account-link-extension/blob/master/build/bundle.js)